README.md

```diff
@@ -0,0 +1,151 @@
+# IntegratorXX
+
+## Synopsis
+
+IntegratorXX is a modern C++ library for the generation of atomic and molecular
+grids for quantum chemistry calculations. It provides a uniform interface for
+the generation of primitive, radial and solid angle quadratures, as well as
+there combination into spherical grids.
+
+## Design Goals 
+
+* Provide stable, reusable, and reproducible implementations of the various atomic and
+molecular grids commonly encountered in quantum chemistry calculations
+* Develop a modern, modular, extensible C++ API to allow for the implementation
+and validation new atomic and molecular quadrature schemes.
+* Provide complete C and Python interfaces to allow reusing the implementation in projects written in these languages
+
+## Dependencies
+
+* CMake (3.17+)
+* Modern C++ compiler (C++17 compliant)
+
+## Major Contributors
+
+* David Williams-Young - LBNL (dbwy at lbl dot gov)
+* Susi Lehtola - University of Helsinki
+
+## Implemented Quadratures
+
+Here we list the quadratures currently implemented in IntegratorXX. Please refer to the
+source for appropriate references.
+
+
+
+### Primitive Quadratures
+
+Primitive quadratures are those generated on a finite bound (e.g. Gauss
+quadrature rules). The general software design pattern of IntegratorXX is to
+build up higher-order quadrature rules (e.g. radial transformation, etc) from
+these primitive quadratures.
+
+| Quadrature Name                 | Domain    | C++ Class           |
+|---------------------------------|-----------|---------------------|
+| Gauss-Chebyshev (First Kind)    | $(-1,1)$  | `GaussChebyshev1`   |
+| Gauss-Chebyshev (Second Kind)   | $[-1,1]$  | `GaussChebyshev2`   |
+| Gauss-Chebyshev (Third Kind)    | $(0,1)$   | `GaussChebyshev3`   |
+| Gauss-Legendre                  | $[-1,1]$  | `GaussLegendre`     |
+| Gauss-Lobatto                   | $[-1,1]$  | `GaussLobatto`      |
+| Trapezoid Rule                  | $[0,1]$   | `UniformTrapezoid`  |
+
+### Radial Quadratures
+
+Radial quadratures are convolutions of primitive quadrature rules with a radial
+transformation scheme (mapping the natural domain of the primitive quadrature
+to positive semi-indefinite). The jacobian of the transformation *is* included
+in the radial weights while the radial component of the spherical jacobian
```

Comment:
should one say "volume element" instead of jacobian here..?

Reply:
That's fair - I typically use them interchangeably as the meaning is to be understood by context, but I'd agree that volume element is the strictly correct term here. Updated